oap-server/server-starter/src/main/resources/meter-analyzer-config/satellite.yaml

```diff
@@ -0,0 +1,34 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+expSuffix: tag({tags -> tags.service = 'satellite::' + tags.service}).instance(['service'], ['instance'])
+metricPrefix: satellite
+metricsRules:
+  - name: instance_receive_event_count
+    exp: gatherer_receive_count.sum(["pipe", "status", "service", "instance"]).increase("PT1M")
```

Comment:
Such as adding sw_stl as prefix in satellite?

Reply:
I see that we already have a filtering mechanism, but it only applies to tags. I think we can add fixed tags to each metric of the satellite and use MAL to filter them. WDYT?

Reply:
been add the `sw_stl ` in satellite metrics service.